cmd/minishift/cmd/start.go

```diff
@@ -179,7 +183,18 @@ func runStart(cmd *cobra.Command, args []string) {
 		atexit.Exit(1)
 	}
 
-	clusterUp(&config)
+	oc, err := clusterUp(&config)
+	if err != nil {
+		glog.Errorln("Error starting the cluster: ", err)
+		atexit.Exit(1)
+	}
+
+	// Update minishift-data.json for oc path
+	minishiftData, err := minishiftdata.Create(machinesDirPath)
```

Comment:
> We want to probably create the initial file in root.go, together with the other data files like config.json.

@hferentschik No, we don't get `machinesDirPath` here since for that it has to be instance of libmachineClient which happens only in start.go 

> root.go actually already creates the machines directory. So imo, the best place to create the instance is there, making sure it gets initialised in the persistent pre-run.

Reason as above.